batch/batch/driver/main.py

```diff
@@ -911,6 +914,116 @@ async def cancel_fast_failing_batches(app):
         await _cancel_batch(app, batch['id'])
 
 
+READY_USER_CORES = pc.Gauge('batch_ready_user_cores', 'Batch ready user cores', ['user', 'inst_coll'])
+RUNNING_USER_CORES = pc.Gauge('batch_running_user_cores', 'Batch running user cores', ['user', 'inst_coll'])
+READY_USER_JOBS = pc.Gauge('batch_ready_user_jobs', 'Batch ready user jobs', ['user', 'inst_coll'])
+RUNNING_USER_JOBS = pc.Gauge('batch_running_user_jobs', 'Batch running user jobs', ['user', 'inst_coll'])
+CREATING_USER_JOBS = pc.Gauge('batch_creating_user_jobs', 'Batch creating user jobs', ['user', 'inst_coll'])
```

Comment:
I think these are better off as
```
USER_CORES … ['state', 'user', 'inst_coll']
````
where `state \in {'ready', 'running'}` and

```
USER_JOBS … ['state', 'user', 'inst_coll']
```
where `state \in {'creating', 'ready', 'running'}`.